### PR TITLE
News about release v2.1.1

### PR DESCRIPTION
**Pull request by baptsegu**

### CODE REVIEW

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
+tags: [RAO, FARAO, release, CORE]
+excerpt_separator: <!--more-->
+---
+
+FARAO v2.1.1 has been recently released. FARAO v2.1.1 notably includes all the featues which were planned in the July's milestone of our [roadmap](/roadmap) <!--more-->, that is to say:
+
+- Optimisation of preventive PST and topological remedial actions
+- Two possible objective functions : min. margin positive or min. margin maximized
+- Branch monitored in one or both directions, with thresholds defined in MW, I or %Imax
```
Comment (phiedw) [inline _posts/2020-08-24-farao-do-the-core-cc.md:14]: in MW, A or %Imax

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Branch monitored in N and N-1 states
+- AC and DC computations (for load flow and sensitivity analyses)
+- Limitation of loopflows in the RAO
+
+With in addition another key feature: the consideration of MNEC (Monitored Network Element) in the optimisation. The Crac importers has been also improved, with some new developpement enabling to efficiently read real data in the standard file formats used for the capacity calculation of CORE, with hundreds of contingencies and thousands of Cnec.
```
Comment (phiedw) [inline _posts/2020-08-24-farao-do-the-core-cc.md:19]: the Crac importers have* also been* improved

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Moreover, before its release, the RAO has been tested on several timestamps of the CORE capacity calculation process. The results of the first remedial action optimisations are very satisfying, and allow to increase the room for power exchanges between CORE countries, while making sure that the loop-flows does not exceed a given threshold, and that the MNECs keep a positive margin.
```
Comment (phiedw) [inline _posts/2020-08-24-farao-do-the-core-cc.md:21]: that the loopflows do* not exceed

**Review by phiedw — CHANGES_REQUESTED** [review]

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: FARAO optimizes the remedial action for the capacity calculation of the CORE region
+category: news
+feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
+tags: [RAO, FARAO, release, CORE]
+excerpt_separator: <!--more-->
+---
+
+FARAO v2.1.1 has been recently released. FARAO v2.1.1 notably includes all the featues which were planned in the July's milestone of our [roadmap](/roadmap) <!--more-->, that is to say:
```
[the remark this answers is not in the document]
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

File: _posts/2020-08-24-farao-do-the-core-cc.md (hunk 1 above)
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

File: _posts/2020-08-24-farao-do-the-core-cc.md (hunk 2 above)
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

File: _posts/2020-08-24-farao-do-the-core-cc.md (hunk 3 above)
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

**Review by phiedw — APPROVED** [review]

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+FARAO v2.1.1 has been recently released. FARAO v2.1.1 notably includes all the features which were planned in July's milestone of our [roadmap](/roadmap) <!--more-->, that is to say:
+
+- Optimisation of preventive PST and topological remedial actions
+- Two possible objective functions : min. margin positive or min. margin maximized
+- Branch monitored in one or both directions, with thresholds defined in MW, A or %Imax
+- Branch monitored in N and N-1 states
+- AC and DC computations (for load flow and sensitivity analyses)
+- Limitation of loopflows in the RAO
+
+With in addition another key feature: the consideration of MNEC (Monitored Network Element) in the optimisation. The Crac importers have also been improved, with some new developpement enabling to efficiently read real data in the standard file formats used for the capacity calculation of CORE, with hundreds of contingencies and thousands of Cnec.
```
Comment (pet-mit) [inline _posts/2020-08-24-farao-do-the-core-cc.md:19]: optimiZation
development

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Moreover, before its release, the RAO has been tested on several timestamps of the CORE capacity calculation process. The results of the first remedial action optimisations are very satisfying, and allow to increase the room for power exchanges between CORE countries, while making sure that the loop-flows do not exceed a given threshold, and that the MNECs keep a positive margin.
```
Comment (pet-mit) [inline _posts/2020-08-24-farao-do-the-core-cc.md:21]: optimiZations

**Review by pet-mit — CHANGES_REQUESTED** [review]

File: _posts/2020-08-24-farao-do-the-core-cc.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+title: FARAO optimizes the remedial action for the capacity calculation of the CORE region
+category: news
+feature-img: "assets/img/Hans_Otto_Theater_Potsdam_-_fake_colors_cut.jpg"
+tags: [RAO, FARAO, release, CORE]
+excerpt_separator: <!--more-->
+---
+
+FARAO v2.1.1 has been recently released. FARAO v2.1.1 notably includes all the features which were planned in July's milestone of our [roadmap](/roadmap) <!--more-->, that is to say:
+
+- Optimisation of preventive PST and topological remedial actions
```
[the remark this answers is not in the document]
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

File: _posts/2020-08-24-farao-do-the-core-cc.md (hunk 5 above)
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

File: _posts/2020-08-24-farao-do-the-core-cc.md (hunk 6 above)
Comment (baptsegu) [reply _posts/2020-08-24-farao-do-the-core-cc.md]: corrected

**Review by pet-mit — APPROVED** [review]

**Pull request merged**